oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/SQLExecutor.java

```diff
@@ -54,4 +54,12 @@ public void invoke(Connection connection) throws SQLException {
         }
         preparedStatement.execute();
     }
+
+    public String getSql() {
+        return sql;
+    }
```

Comment:
You could override `hashcode` and `equals` methods using SQL as the only condition. Then, this object could be taken as `key` of HashMap. We don't need another `getSql` method actually, from my understanding.

Reply:
ok .done